### DISCUSSION TITLE
Far и far2l: сравнение функциональности «горячих» клавиш

### DISCUSSION THREAD

**Discussion by Mischutka** [elfmz/far2l#1158] [Q&A]
Разместил [здесь](https://www.linux.org.ru/forum/desktop/16705955) небольшой перечень "горячих" клавиш FAR с результатами проверки их работоспособности в far2l, вдруг кому-то данная информация окажется полезной.

Убедительная просьба к пользователям far2l, проверьте у себя, пожалуйста, работу комбинации клавиш Ctrl+M, дабы убедиться, что восстановление последнего выделения файлов не работает вообще, а не только у меня.

**Comment by Mischutka** [comment]
Поскольку раздел называется "Вопросы и ответы", то сформулирую так: работает ли у вас в far2l комбинация клавиш Ctrl+M в качестве восстановления последнего выделения файлов?

**Reply by unxed** [reply]
У меня сработало. wx-версия

**Reply by singalen** [reply]
MacOS, WX - Cmd-M работает.

**Comment by elfmz** [comment]
В GUI режиме работает, в терминале - нет. С сегодняшними изменениями работает и в терминале, если доступны иксы и far2l собран с libx11-dev и libxi-dev.

**Reply by Mischutka** [reply]
Благодарю. Если не секрет, то каким образом проводилось выделение файлов? [Как выяснилось, способ выделения файлов имеет значение.](https://forum.astralinux.ru/threads/2049/post-17562)

**Reply by elfmz** [reply]
кнопкой Ins

**Reply by elfmz** [reply]
Прочитал последнее сообщение по вашей ссылке 'Я делал иным способом: выделил мышью файл, затем снял выделение мышью, а потом пробовал восстановить его нажатием Ctrl+M.'
Так - не работает, и вроде не должно. Во всяк случае поведение в 3м виндовом фаре идентичное - Ctrl+M снимает восстанавливает выделенные файлы, а если снять выделение сторонним образом Ctrl+М его не вспоминает

**Comment by unxed** [comment]
Раз тема про сравнение, пожалуй, упомяну тут проблему с Ctrl+<.> на русской раскладке. И ещё с несколькими соседними кнопками - теми, где нет английских букв, а есть пунктуация.

Детали тут
https://github.com/elfmz/far2l/issues/1211

**Comment by martid1703** [comment]
Привет, подскажите, как сделать так чтобы при нажатии Alt+F1 срабатывал ФАР, а не селектор воркспейса? (Linux Green)

**Reply by Mischutka** [reply]
Отключить в настройках "горячих" клавиш Linux Green сочетание Alt+F1 для запуска селектор воркспейса.

**Reply by martid1703** [reply]
помогло, спасибо)
